homestar-runtime/src/event_handler/event.rs

```diff
@@ -457,8 +562,8 @@ impl Replay {
                                 SwarmNotification::PublishedReceiptPubsub,
                             ),
                             btreemap! {
-                                "cid" => receipt.cid().to_string(),
-                                "ran" => receipt.ran().to_string()
+                                "cid" => Ipld::String(receipt.cid().to_string()),
```

Comment:
Yeah, this will all change once we have implemented JSON Schema. Instead, we'll be instantiating structs for each notification. Will leave as is and keep this in mind in the schema work. 👍